packages/nextjs/src/config/loaders/dataFetchersLoader.ts

```diff
@@ -151,22 +151,33 @@ export default function wrapDataFetchersLoader(this: LoaderThis<LoaderOptions>,
     if (hasDefaultExport(ast)) {
       outputFileContent += `
         import { default as _sentry_default } from "${this.resourcePath}?sentry-proxy-loader";
-        import { withSentryServerSideGetInitialProps } from "@sentry/nextjs";`;
+        import {
+          withSentryServerSideGetInitialProps,
+          withSentryServerSideAppGetInitialProps,
+          withSentryServerSideDocumentGetInitialProps,
+          withSentryServerSideErrorGetInitialProps,
+        } from "@sentry/nextjs";`;
 
       if (parameterizedRouteName === '/_app') {
```

Comment:
Once this is moved over to the proxy template, what would think of using a map for all of these checks? Something like

```js
const getInitialPropsWrappers = {
  "/_app": Sentry.withSentryServerSideAppGetInitialProps,
  "/_document": Sentry.withSentryServerSideDocumentGetInitialProps,
  "/_error": Sentry.withSentryServerSideAppGetInitialProps,
}

const getInitialPropsWrapper = getInitialPropsWrappers["__ROUTE__"] || Sentry.withSentryServerSideErrorGetInitialProps

if (typeof origGetInitialProps === 'function') {
  pageComponent.getInitialProps = getInitialPropsWrapper(
    origGetInitialProps,
    '__ROUTE__',
  ) as NextPageComponent['getInitialProps'];
}
```

Reply:
If we merge https://github.com/getsentry/sentry-javascript/pull/5602 I will rebase this PR and also https://github.com/getsentry/sentry-javascript/pull/5593 onto it so that the wrappers use the new proxy loader. This seems like a good approach! 👍